### PR TITLE
Fix space around operators

### PR DESCRIPTION
**Pull request by tatyree**
As per the .rubocop_todo.yml.  The definition of this one seems unproblematic.  Merge away if you're happy...

### CODE REVIEW

File: app/controllers/ping_controller.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   def index
     render json: {
-      :version_number => VERSION_INFO['version_number'],
-      :build_date => VERSION_INFO['build_date'],
-      :commit_id => VERSION_INFO['commit'],
-      :build_tag => VERSION_INFO['build_tag']
+      version_number:  VERSION_INFO['version_number'],
+      build_date:      VERSION_INFO['build_date'],
+      commit_id:       VERSION_INFO['commit'],
+      build_tag:       VERSION_INFO['build_tag']
```
Comment (SteveMarshall) [inline app/controllers/ping_controller.rb]: :+1:

Lining up FTW.

Comment (tatyree) [inline app/controllers/ping_controller.rb]: I gots me a plugin to do it real purty like :)

File: spec/shared_examples_prison_slots_and_data.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+RSpec.shared_examples "slots and data" do
```
Comment (SteveMarshall) [inline spec/shared_examples_prison_slots_and_data.rb]: This is a nice refactoring-out of these :stars:

Might be nice if the name had `prison` in it, though?

Comment (tatyree) [inline spec/shared_examples_prison_slots_and_data.rb]: Thank you.

Hmmm, it did at one point.  I must have cut it out by accident.  I'll put
it back in.

**Comment by tatyree** [comment]
@SteveMarshall 'prison' is now there...

**Pull request merged**